rpcs3/Emu/system_progress.cpp

```diff
@@ -252,6 +253,10 @@ void progress_dialog_server::operator()()
 						{
 							fmt::append(progr, " (%um remaining)", minutes);
 						}
+						else if (minutes == 0)
+						{
+							fmt::append(progr, " (%02us remaining)", seconds);
```

Comment:
But the minutes being 0 helps the user notice that it is seconds.

Reply:
Did you just assume the users can't read?

Reply:
In seriousness, this just looks weird and does not provide any meaningful information.
Any designer will tell you to drop it

Reply:
I assume that is a single letter difference is not very noticeable and that the MM:SS format is very recognizable.